egs/iwslt22_ta/ST/local/transcript_cleaning.py

```
@@ -50,194 +51,266 @@
-    if word.replace('.','').isnumeric():  # remove the dot if it is not part of a number 
-      res.append(word)
+    words = text.split()
+    res = []
+    for word in words:
+        if word.replace(
+            ".", ""
+        ).isnumeric():  # remove the dot if it is not part of a number
+            res.append(word)
+
+        else:
+            word = re.sub("\s+\.", "", word)
```

Ruff (0.15.18)

[warning] Invalid escape sequence: `\s`

Use a raw string literal

(W605)